matlab/Ice/context/client.m

```diff
@@ -0,0 +1,59 @@
+% Copyright (c) ZeroC, Inc.
+
+function client(args)
+    % We put the code generated by the Slice compiler in the 'generated' subdirectory.
+    addpath('generated');
+
+    % Load the Ice library if it is not already loaded.
+    if ~libisloaded('ice')
```

Comment:
I guess we use this model because we don't call `unloadlibrary` we probably should.

Reply:
For a follow-up PR.